pennylane_lightning/core/src/simulators/lightning_tensor/tncuda/measurements/MeasurementsTNCuda.hpp

```diff
@@ -164,6 +165,98 @@ template <class TensorNetT> class MeasurementsTNCuda {
         return h_res;
     }
 
+    /**
+     * @brief Utility method for samples.
```

Comment:
```suggestion
     * @brief Samples the discrete random distribution underlying an MPS state.
```

Reply:
Let's keep current brief since this API works for both MPS and ExaTN.